README.md

```diff
@@ -49,10 +49,147 @@ $ yarn lint
 # Run tests
 $ yarn test
 
-# Record new trafic with live API calls
+# Record new trafic with live API calls, more information bellow
 $ yarn test:record grid/path/to/test.ts
 ```
 
+Jest in station is configured to run tests inside `/grid` folder by default, with reporter located in `/jest/reporter` and can use groups to filter which tests to run (mainly used for tagging tests ready to continuous testing).
```

Comment:
I wanted to add overview of test configurations that station has/will have, since with one I added in #263 there would be 3 `jest.config.js` files, but totally failed describing it 😄 

I can remove this and describe it under relevant commands and later add this section after #263 is merged 🤷🏻‍♂️

Reply:
Yeah, Ondra explained to me that this section will be moved to the contribution guide, so it probably makes more sense there. I'd dedicate a separate section there to configuration files.

Reply:
Ok, I'll remove this with example